newswires/app/conf/Database.scala

```diff
@@ -0,0 +1,113 @@
+package conf
+
+import play.api._
+import scalikejdbc._
+import software.amazon.awssdk.services.ssm.SsmClient
+import software.amazon.awssdk.services.ssm.model.GetParameterRequest
+import software.amazon.jdbc.ds.AwsWrapperDataSource
+
+import java.util.Properties
+
+object Database extends Logging {
+
+  def healthcheck = {
+    DB localTx { implicit session =>
+      sql""" SELECT 1 as successes """
+        .map(_.int("successes"))
+        .single()
+        .apply()
+        .get
+    }
+  }
+
+  // def getDevDbConfig(initialConfiguration: Configuration): Configuration = {
+  //   logger.info("building DB config for DEV")
+  //   val username = initialConfiguration
+  //     .get[String]("database.username")
+  //   val port =
+  //     initialConfiguration.get[Int]("database.port")
+  //   val hostname = initialConfiguration.get[String]("database.endpoint-address")
+  //   val password = initialConfiguration
+  //     .get[String]("database.password")
+  //
+  //   val configuration =
+  //     Map(
+  //       "user" -> username,
+  //       "portNumber" -> port,
+  //       "serverName" -> hostname,
+  //       "databaseName" -> "birthdays",
+  //       "password" -> password
+  //     )
+  //
+  //   Configuration.from(
+  //     Map(
+  //       "slick.dbs.default.db.properties" -> configuration
+  //     )
+  //   )
+  // }
```

Comment:
is this intentionally commented with the intention of reinstating soon for local postgres access (i.e. displaying what ingestion lambda is producing)

Reply:
yep exactly